apps/sms/index.html

```diff
@@ -128,7 +129,7 @@
 
   <body role="application" class="skin-comms theme-communications">
     <article id="main-wrapper" class="wrapper" data-position='right'>
-      <section id="thread-list" class="panel default-panel" role="region">
+      <section class="panel panel-thread-list" role="region">
```

Comment:
Using `panel-` is fine for me. And we might need to polish the panel name for navigation in the future like Oleg said.